lib/doc2pdf.rb

```diff
@@ -10,7 +10,7 @@ module Doc2pdf
   class Error < StandardError; end
 
   # Example: "foo {bar} spam {egg} asd" -> ['bar', 'egg']
-  PLACEHOLDER_PATTERN = /{(\w*)}/.freeze
+  PLACEHOLDER_PATTERN = /({\w*})/.freeze
```

Comment:
Now I get why you want to capture also `{` and `}`. It's that so you can replace the whole string later.
Gotcha! Please update the comment above then.